src/aap_eda/services/auth.py

```diff
@@ -29,3 +34,17 @@ def display_permissions(role_data: dict) -> dict:
     grouped_permissions = group_permission_resource(role_data["permissions"])
     role_data["permissions"] = grouped_permissions
     return role_data
+
+
+def create_jwt_token() -> tuple[str, str]:
+    """Create JWT access and refresh token pair.
+
+    They can be sent to rulebook clients through command line arguments.
+    """
+    user, _ = User.objects.get_or_create(
+        username="_service_user",
+        is_service_account=True,
+        defaults={"password": uuid.uuid4()},
```

Comment:
@bzwei the python secrets package has functions for this https://docs.python.org/3/library/secrets.html#recipes-and-best-practices